### PR TITLE
update instructions to reflect changes in cp eyebrow

### PR DESCRIPTION
**Pull request by stephamon**
full audit for card INFODEV-1394

### CODE REVIEW

**Comment by cyrichardson** [comment]
Just a couple of tiny nits, @
[viewing-and-editing-user-information-for-your-account_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1938449/viewing-and-editing-user-information-for-your-account_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@stephamon Edits in the file above.

**Comment by cyrichardson** [comment]
Oopps! Sorry, @stephamon! Hit the wrong button.

File: content/general/viewing-and-editing-user-information-for-your-account.md [hunk 1]
```diff
@@ -1,50 +1,47 @@
...
-
--   To add a new user, click **Create User**. On the **Create
-    User** page, enter information for the new user's login, access
-    levels, and contact information. For information about access
-    controls for new users, see [Learning about Role-Based Access Control (RBAC)](/how-to/overview-role-based-access-control-rbac).
-    After you have entered all of the information for the new user,
-    click the **Create User** button at the bottom of the page.
+- To make changes for a user, click the user name. The **My Profile & Settings** page displays. You can edit contact information, security settings, and account and product permissions.
+
+- To add a new user, click **Create User**. On the **Create User** page, enter information for the new user's login, contact information, security question, and access level. For information about access controls for new users, see [Learning about Role-Based Access Control (RBAC)](/how-to/overview-role-based-access-control-rbac). After you have entered all of the information for the new user, click the **Create User** button at the bottom of the page.
```
Comment (RackerKate) [inline content/general/viewing-and-editing-user-information-for-your-account.md]: The linked page is now called "Learn about..." instead of "Learning...".

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9b446a6bff/how-to/)

**Pull request merged**